### DISCUSSION TITLE
How to use duplicate tittle value

### DISCUSSION THREAD

**Discussion by Trung15010802** [imaNNeo/fl_chart#1592] [Q&A]
I want nana show my x axis like: 1 2 3 1 2 3
But it's only show from 1 to 3. How can i achieve that?
![image](https://github.com/imaNNeo/fl_chart/assets/90593296/b8650db1-dc09-4925-851e-27e782b3f543)

**Comment by imaNNeo** [comment]
I didn't understand what exactly you wanted to achieve
